app/src/main/java/com/github/gotify/service/WebSocketService.java

```diff
@@ -21,18 +23,32 @@
 import com.github.gotify.R;
 import com.github.gotify.Settings;
 import com.github.gotify.Utils;
+import com.github.gotify.api.CertUtils;
 import com.github.gotify.api.ClientFactory;
+import com.github.gotify.client.ApiClient;
+import com.github.gotify.client.api.ApplicationApi;
 import com.github.gotify.client.api.MessageApi;
+import com.github.gotify.client.model.Application;
 import com.github.gotify.client.model.Message;
 import com.github.gotify.log.Log;
 import com.github.gotify.log.UncaughtExceptionHandler;
 import com.github.gotify.messages.Extras;
 import com.github.gotify.messages.MessagesActivity;
+import com.github.gotify.messages.provider.MessageImageCombiner;
+import com.squareup.picasso.OkHttp3Downloader;
+import com.squareup.picasso.Picasso;
+import com.squareup.picasso.RequestCreator;
+
+import java.io.File;
+import java.io.IOException;
 import java.util.List;
 import java.util.Map;
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.atomic.AtomicInteger;
 
```

Comment:
The travis-ci build currently fails because of formatting errors. Run `./gradlew spotlessApply` to fix them.
https://travis-ci.org/github/gotify/android/builds/677337457#L1868